### DISCUSSION TITLE
HTTPS Support

### DISCUSSION THREAD

**Discussion by paromarc** [AxisCommunications/media-stream-library-js#756] [Q&A]
Does the media-stream-library support https ? If not, do we have an alternative available?

Thanks!

**Comment by paromarc** [comment]
I think I figured it out :-) 
I just changed a couple of lines in simple_player.js.

fecth login using https and using wss instead of ws

**Comment by Glenn-io** [comment]
@paromarc I would absolutely LOVE to know how you were able to get simple_player.js and media-stream-library.min.js to work over https. We have an internal web application that we can't force https since the cameras only work on http. Thank you in advance!
